### ISSUE TITLE
<linux/serial.h> AND <SDL/SDL.h> Missing on Mac

### ISSUE THREAD

**Issue by alexmead** [ComputerNerd/simpleFrameGrabber#1]
Could you please comment on how to make this code compile on Mac OS X?  It seems you use two platform dependent header files (I think this code is would compile on Linux no problem) and I'm having trouble figuring out how to get it to compile on Max OS X.
Thanks!

**Comment by ComputerNerd** [comment]
Yes linux/serial.h is platform dependent. The need for linux/serial.h is removed for systems without it in commit 1af0cd10afc4db19b07bc1a40fc5b6aee4531d84. For SDL/SDL.h you need to install libSDL 1.2 if you have not already done so. If that does not solve the problem find where SDL.h is.

I have not tried compiling on Mac OS X. It compiled by adding #undef __linux__ which simulated an attempted to compile on a system that does not run Linux, the kernel. If you still cannot get it to work be prepared to modify my code for Mac OS X support.

**Comment by abingham** [comment]
I managed to compile this on macos with:
```
clang -fobjc-nonfragile-abi -o grab main.c -lSDL -lSDLmain -ljpeg -lobjc -framework Cocoa
```
I don't know if it's actually working yet, but I suspect any problems I'm having are on the camera end.
